### DISCUSSION TITLE
Welcome to timewarrior Discussions!

### DISCUSSION THREAD

**Discussion by lauft** [GothenburgBitFactory/timewarrior#431] [General]
## 👋 Welcome!
Timewarrior Discussions is a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by rafalkrupinski** [comment]
Finally, time tracking for geeks
AWESOME!
